app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+from models import Actor

```

Comment:
needs to be a blank line between different kinds of imports + we use full path from app